<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Message">
    <attribute name="title">
     <string>Message</string>
    </attribute>
    <widget class="QTextEdit" name="messageText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>651</width>
       <height>491</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonSend">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>520</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonClear">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="widgetSelectChannel" native="true">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>430</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelNameAddPaste">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Marker Felt</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>          paste</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="BoxPaste">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addPaste">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widgetSleep" native="true">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>70</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <widget class="QSpinBox" name="spinBoxSleep">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Impact</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
     </widget>
     <widget class="QLabel" name="labelSleep">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Marker Felt</family>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>    sleep</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widgetAddPaste" native="true">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>340</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelNameSelectChannel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Marker Felt</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>   select channel</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="BoxChannel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="sizeText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>480</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>symbols  0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSplit">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Marker Felt</family>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>        split</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="addSybSplit">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="options">
    <attribute name="title">
     <string>Options</string>
    </attribute>
    <widget class="QWidget" name="widgetAccount" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>571</height>
      </rect>
     </property>
     <widget class="QTreeWidget" name="treeWidgetAcc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>291</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Nickname</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>authorization</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="ButtonAddAcc">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>520</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonDeleteAcc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNameAcc">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Marker Felt</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Accounts</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widgetChannel" native="true">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>301</width>
       <height>571</height>
      </rect>
     </property>
     <widget class="QTreeWidget" name="treeWidgetChanel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>291</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>id channel</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="ButtonAdd_Channel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>520</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonDeleteChannel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNameChannel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Marker Felt</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Channels</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widgetPaste" native="true">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>0</y>
       <width>301</width>
       <height>571</height>
      </rect>
     </property>
     <widget class="QTreeWidget" name="treeWidgetPaste">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>291</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Paste</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="ButtonAddPaste">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>520</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonDeletePaste">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNamePaste">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Marker Felt</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>  Paste</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
